app/assets/javascripts/directives/human_time.js

```diff
@@ -0,0 +1,10 @@
+angular.module("OpenHq").directive("humanTime", function() {
+    return {
+        scope: {
```

Comment:
not sure yet if it should be put in, but any examples I saw of directive they always put a `restrict: "E",` in above scope. This restricts its usage to `<human-time>` rather than `<div human-time>` or `<div class="human-time">`. The attribute one is `A` and the class name one is `C` in the restrict. Probably a good idea even for our own documentation to put a restrict in with how we think it should be used.